src/Analyzers/CSharp/Analyzers/UseIndexOrRangeOperator/CSharpUseRangeOperatorDiagnosticAnalyzer.cs

```diff
@@ -165,8 +166,17 @@ protected override void InitializeWorker(AnalysisContext context)
             // Second arg needs to be a subtraction for: `end - e2`.  Once we've seen that we have
             // that, try to see if we're calling into some sort of Slice method with a matching
             // indexer or overload
+            var valid = infoCache.TryGetMemberInfo(targetMethod, out var memberInfo);
```

Comment:
why was this extracted?

Reply:
Just fixed, I had been using valid for something else later but then removed it